ci/infra/testrunner/tests/conftest.py

```diff
@@ -11,9 +11,18 @@ def pytest_addoption(parser):
     """
     parser.addoption("--vars", action="store",help="vars yaml" )
     parser.addoption("--platform", action="store",help="target platform" )
+    parser.addoption("--skip-setup",
+                     choices=['provisioned', 'bootstrapped', 'deployed'],
```

Comment:
I would move this to the test runner so that this is visible to the testrunner's help